keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -287,14 +270,22 @@ class ClusterHandler(
       }
         .mapNotNull { it.await() }
         .toSet()
+//        .also { them ->
```

Comment:
Looks like this can be fully removed?

Reply:
That's actually the change I want to make next. Didn't mean to commit it.